cli/src/base.ts

```diff
@@ -42,20 +44,26 @@ export abstract class BaseCommand extends Command {
   // In the future we can support YAML
   searchPlaces = [`.${moduleName}rc`, `.${moduleName}rc.json`, 'package.json'];
 
-  static databaseURLFlag = Flags.string({
-    description: 'URL of the database in the format https://{workspace}.xata.sh/db/{database}'
-  });
+  static databaseURLFlag = {
+    db: Flags.string({
+      helpValue: 'https://{workspace}.xata.sh/db/{database}',
+      description: 'URL of the database in the format https://{workspace}.xata.sh/db/{database}'
```

Comment:
I find the format redundant right now. We show it both in the description and help value. 


```suggestion
      helpValue: 'https://{workspace}.xata.sh/db/{database}',
      description: 'URL of the database'
```